nodepy/runge_kutta_method.py

```diff
@@ -2587,6 +2589,8 @@ def loadRKM(which='All'):
           * 'DP5':        Dormand-Prince 5th-order :cite:`dormand1980`
           * 'PD8':        Prince-Dormand 8th-order and 7th-order pair :cite:`prince1981`
           * 'CMR6':       Calvo et al.'s 6(5) pair :cite:`calvo1990`
+          * 'Zonneveld43': 4(3) pair of Zonneveld :cite:`hairer1993` Table 4.2
+          * 'Söderlind43': 4(3) pair of Söderlind
```

Comment:
I already merged this but the second character makes it rather challenging to load this method on a non-german keyboard.  Perhaps better to go with convenience over perfect spelling in this situation.

Reply:
Maybe you're right. Although it is not really hard to do (I'm using an English keyboard right now). In Linux, there is a compose key to write such characters. 
Shall I rename Söderlind43 to Soderlind43?

Reply:
>Shall I rename Söderlind43 to Soderlind43?

Yes, I think it's for the best and that Gustaf will forgive us. :)